Healthy/Classes/Modules/Dashboard/Views/SliderDishesView/SliderDishesView.swift

```diff
@@ -0,0 +1,79 @@
+import UIKit
+
+final class SliderDishesView: UIView {
+
+    // MARK: - Outlet
+
+    @IBOutlet private(set) weak var sliderView: UIView!
+    @IBOutlet private(set) weak var collectionView: UICollectionView!
+
+    // MARK: - Properties
+    private var viewModel: [SliderCollectionViewCell.ViewModel] = [
+        // TODO: To be removed when real data are integrated
+        .init(imageUrl: UIImage(named: "preview-dishes-1"), dishName: "", time: "", duration: .zero),
+        .init(imageUrl: UIImage(named: "preview-dishes-2"), dishName: "", time: "", duration: .zero),
+        .init(imageUrl: nil, dishName: "", time: "", duration: .zero),
+        .init(imageUrl: nil, dishName: "", time: "", duration: .zero)
+    ]
+
+    // MARK: - initializer
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        configureCollectionView()
+
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+
+        configureCollectionView()
+    }
+}
+
+// MARK: - CollectionView setup
+
+private extension SliderDishesView {
+
+    func configureCollectionView() {
+        loadViewFromNib()
+        collectionView.register(SliderCollectionViewCell.self)
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        configureCollectionViewLayout()
+    }
```

Comment:
```suggestion
    override init(frame: CGRect) {
        super.init(frame: frame)

        loadViewFromNib()
        configureCollectionView()
    }

    required init?(coder: NSCoder) {
        super.init(coder: coder)

        loadViewFromNib()
        configureCollectionView()
    }
}

// MARK: - CollectionView setup

private extension SliderDishesView {

    func configureCollectionView() {
        collectionView.register(SliderCollectionViewCell.self)
        collectionView.delegate = self
        collectionView.dataSource = self
        configureCollectionViewLayout()
    }
```

Loading view from nib is not a view's configurations.